### ISSUE TITLE
Create Weave (Compose) Fluent API

### ISSUE THREAD

**Issue by mariotoffia** [mariotoffia/FluentDocker#9]
This should support both fluent instructions to weave containers and networks as well as providing yaml files for docker-compose. If docker-compose is used those containers shall be reflected into a weave service instance (mesh) and be controllable from those.

**Comment by lorenzo93** [comment]
I also need to use docker-compose commands providing yml file.
@mariotoffia Do you have a timeline for the release?

**Comment by mariotoffia** [comment]
@lorenzo93 It will be sometime this winter. In meantime you *can* do _docker-compose_. For example:
```
      var file = "docker-compose-file.yml";
      // "Execute" compose file 
      Host.Host.ComposeUp(composeFile: file, certificates: Host.Certificates);

      // Use this to get hold on get containers. You can use container discover as well.
      var containers = Host.Host.ComposePs(composeFile: file, certificates: Host.Certificates);

      // You have now a running system from compose
      
      // Tear down
      Host.Host.ComposeDown(composeFile: file, certificates: Host.Certificates, removeVolumes: true,
          removeOrphanContainers: true);
```

Cheers,
 Mario

**Comment by mariotoffia** [comment]
@lorenzo93 There now is a fluent _API_ to do docker-compose. However it is in early alpha but it is included in the 2.4.0. There's some documentation at https://github.com/mariotoffia/FluentDocker#docker-compose-support

Cheers, 
 Mario
